Make minor language improvement/fix punctuation (#17388)
mdn/content: files/en-us/learn/javascript/asynchronous/index.md

## Guides

- [Introducing asynchronous JavaScript](/en-US/docs/Learn/JavaScript/Asynchronous/Introducing)

>   - : In this article, we learn about **synchronous** and **asynchronous** programming, why we often need to use asynchronous techniques, how asynchronous functions have historically been implemented in JavaScript, and what their problems are.

Reviewer: ```suggestion
  - : In this article, we'll learn about **synchronous** and **asynchronous** programming, why we often need to use asynchronous techniques, and the problems related to the way asynchronous functions have historically been implemented in JavaScript.
```

Author: Yes, much easier to understand.

- [How to use promises](/en-US/docs/Learn/JavaScript/Asynchronous/Promises)

>   - : We introduce promises and show how to use promise-based APIs. We also introduce the `async` and `await` keywords.

Reviewer: ```suggestion
  - : Here we'll introduce promises and show how to use promise-based APIs. We'll also introduce the `async` and `await` keywords.
```
The original text is good.

- [Implementing a promise-based API](/en-US/docs/Learn/JavaScript/Asynchronous/Implementing_a_promise-based_API)

>   - : We outline how to implement your own promise-based API.

Reviewer: ```suggestion
  - : This article will outline how to implement your own promise-based API.
```
Keeping future tense to be consistent with other descriptions on this page.

- [Introducing workers](/en-US/docs/Learn/JavaScript/Asynchronous/Introducing_workers)

>   - : Workers enable you to run certain tasks in a separate thread to keep your main code responsive. In this article, we rewrite a long-running synchronous function to use a worker.

Reviewer: ```suggestion
  - : Workers enable you to run certain tasks in a separate thread to keep your main code responsive. In this article, we'll rewrite a long-running synchronous function to use a worker.
```
